src/styles/variables.ts

```diff
@@ -151,7 +151,7 @@ export default {
     pressDimValue: 0.8,
     qrShareHorizontalPadding: 32,
 
-    baseMenuItemHeight: 64,
-
     moneyRequestSkeletonHeight: 107,
+
+    distanceScrollEventThrottle: 16,
```

Comment:
NAB: this might be better in CONST because it's not strictly about styles, but I'm not too concerned about it